AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -991,18 +991,18 @@ - (NSArray *)completedNodes
 - (void)dealloc
 {
   ASDisplayNodeAssertMainThread();
-  [_completedNodes enumerateKeysAndObjectsUsingBlock:^(NSString *kind, NSMutableArray *nodes, BOOL *stop) {
-    [nodes enumerateObjectsUsingBlock:^(NSMutableArray *section, NSUInteger sectionIndex, BOOL *stop) {
-      [section enumerateObjectsUsingBlock:^(ASCellNode *node, NSUInteger rowIndex, BOOL *stop) {
+  [_completedNodes enumerateKeysAndObjectsUsingBlock:^(NSString *kind, NSMutableArray *sections, BOOL *stop) {
```

Comment:
@eanagel Hi Ethan, do we still need to remove all the layer / views from the node in here? Why did we added that in the first place?

cc @appleguy